docker/realm.json

```diff
@@ -0,0 +1,1803 @@
+{
```

Comment:
Are those settings reviewed by someone in regards to security? (verifyEmail, token lifespans, etc.)?

Reply:
These are just test data.